pyeudiw/tests/federation/test_static_trust_chain_validator.py

```diff
@@ -163,6 +163,7 @@
             "homepage_uri": "https://verifier.example.org/home",
             "policy_uri": "https://verifier.example.org/policy",
             "logo_uri": "https://verifier.example.org/static/logo.svg",
+            "federation_fetch_endpoint": "https://verifier.example.org/fetch",
```

Comment:
No, a Leaf cannot have a fetch endpoint

the fetch endpoint MUST be only exposd by a Trust Anchor and its Intermediates

Reply:
Resolved in [fix: fixed test test_update_st_es_case_source_endpoint](https://github.com/italia/eudi-wallet-it-python/pull/77/commits/8ce2aaed1759dbce9146541bfdad7493317e8f18)